tests/kuttl/tests/fernet_rotation/03-rotate_keys.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keystone
+  annotations:
+    keystone.openstack.org/rotatedat: "2009-11-10T23:00:00Z"
+commands:
+  - script: |
+      #!/bin/sh
```

Comment:
This script is repeated after every secret. Maybe create a script with this content and just call the script to keep it more clear and avoid redundancy?

Reply:
Yea, agreed, I was waiting for it to pass the tests to refactor, but that's minor change, I'll add that and `validate_test_token.sh` and `test_invalid_token.sh` since they are both basically the same.